website/README.md

```diff
@@ -48,6 +48,8 @@ All images and other static assets you wish to include should be placed inside t
 
 Files placed in `static/` will be accessible in the following way: `static/img/image.png` will be accessible at `img/image.png`.
 
+For new SVG images execute the `yarn svgo path/to/image.svg` (or `yarn svgo-all` if you want to normalize them all) before adding them to the repo.
+
```

Comment:
If anyone can help me with words here would be great :)

Reply:
> Please run <code>yarn svgo *path/to/image.svg*</code> before committing a new SVG to the repository.

How about that?